### ISSUE TITLE
BibCirculation: holdings do not display due to hardcoded collection requirement

### ISSUE THREAD

**Issue by aw-bib** [inveniosoftware/invenio#2592]
In websearchadminlib, around line 3527 (https://github.com/inveniosoftware/invenio/blob/master/modules/websearch/lib/websearchadminlib.py#L3527) one finds the following code:

```
#Disable holdings tab if collection != Books
collection = run_sql("""select name from collection where id=%s""", (colID, ))
if collection[0][0] != 'Books':
    tabs['holdings']['enabled'] = False
```

This effectively hides the _contents_ of the _holdings_ tab in case the collection is not called `Books` regardless of the status of the tab itself. That is, the tab can well be visible, but still will not contain anything.

I would suggest, as already stated in the same module some lines higher up, to remove this block or (as @tiborsimko suggested) at least hide it in an if CFG_SOMESITE block if needs be. OTOH, if one doesn't want to have holdings to be displayed one can (now) easily disable the tab itself through websearchadmin interface, so there seems to be no reason to explicitly hide the contents but still display the tab.

Additionally, many library catalogues handle contents beyond books, so the naming is also a bit counterintuitive. (Something like OPAC would be nearer the mark.)

Please strongly consider also backporting this fix to 1.1 maint.

**Comment by tiborsimko** [comment]
Yes, the code dates from 2008 and can go, provided one fills `collectiondetailedrecordpagetabs` properly for all collections, ideally with an upgrade recipe.

**Comment by aw-bib** [comment]
This issue is closed but not backported to the 1.1 maint. Is this intended?

**Comment by aw-bib** [comment]
If I am not mistaken this fix shows only up in 1.2 not in 1.1.
